### ISSUE TITLE
Common: Model selection

### ISSUE THREAD

**Issue by MatsM16** [MatsM16/uib.inf265.project2#1]
# Model selection
We need a simple readable way to do model selection.  
This code should be usable on both `Task 2` and `Task 3`.

## Required behaviour
1. Define and train some models
2. Validate all the models on a given validation set
3. Pick the best model
4. Test performance on test set.

## Nice to have
- Log some data so we can generate graphs for the report.
